book/models/GP_EllipticalSliceSampler.md

```diff
@@ -55,27 +52,27 @@ import numpy as np
 from blackjax import elliptical_slice, nuts, window_adaptation
 ```
 
-```{code-cell} python
+```{code-cell} ipython3
 def squared_exponential(x, y, length, scale):
     dot_diff = jnp.dot(x, x) + jnp.dot(y, y) - 2 * jnp.dot(x, y)
     return scale**2 * jnp.exp(-0.5 * dot_diff / length**2)
 ```
 
-```{code-cell} python
-def inference_loop(rng, init_state, kernel, n_iter):
+```{code-cell} ipython3
+def inference_loop(rng, init_state, step_fn, n_iter):
```

Comment:
either keep `kernel`, or use `step_fn` throughout (you named them `step` below)

Reply:
Thanks for the suggestion. I have added in the `_fn` suffix on `init` and `step`. I think that is consistent with the naming of `step_fn` and `kernel` in the NUTS and HMC source code? My understanding is that `step_fn` is similar to `kernel` but fixing all arguments that doesn't change with iterations?

I have also prefixed `es_` and `nuts_` to make it clearer.